content/client-side-forms.md

```diff
@@ -5,7 +5,32 @@ metaDescription: "sensenet Client Side Forms"
 ---
 
 # Client Side Forms
+
+As you probably heard before, in sensenet everything is a content. This philosophy provides many advantages when working with sensenet because the only difference between contents are the fields they have. This facilitate the way of managing different [type of contents](/concepts/content-types).
+
+Based on this logic, defining a new content type with custom fields, opens the possibility to create client side forms easily.
+
 # Schemas and Forms
+
+You have the newly defined content type (form) but how this will be filled by users, you may ask.
+Every time a user fills the fields (questions) of this content and submit it, a new instance is created (filled form) containing all the answers given to the questions in the form (saved to the corresponding fields).
+
+Schemas
+???
```

Comment:
Field on content types can have different types and different configurations just like questions may have on a survey. So if you have your questions you can easily select the matching types using a wide variety of field types from simple text and number fields till more complex ones containing multiple options to choose or list of links as answers. The fields can also be used to refine your questions with setting minimum or maximum values, set whether a question/field is required to fill or adding more complex rules for example for working with email addresses or phone numbers as answers and many more.

Schema represents the content model of the repository, the content types you create. This schema contains all the types, their fields and field configs and could be downloaded from your repository on the fly (through OData REST API) that lets you build custom ui for your form.